### ISSUE TITLE
Seems jenkins changed a bunch of links

### ISSUE THREAD

**Issue by pmoosh** [ICTO/ansible-jenkins#42]
- name: Get Jenkins updates
  get_url: url=http://updates.jenkins.io/update-center.json
- name: Get Jenkins updates without proxy
  get_url: url=https://updates.jenkins.io/current/update-center.json 
- name: Import jenkins key
  rpm_key:
    key: http://pkg.jenkins-ci.org/redhat/jenkins-ci.org.key

And maybe a few more ...

 I guess this all due to Jenkins 2.0 being released. Looks like there is a lot more changes...

Was probably intermittent - and the old links work again ....

**Comment by kevinharvey** [comment]
Relatedly, I'm getting the following error trying to install on a fresh Ubuntu 14.04 Vagrant box:

```
TASK [flyapen.jenkins : Get Jenkins updates] ***********************************
fatal: [127.0.0.1]: FAILED! => {"changed": false, "failed": true, "msg": "SSL Certificate does not belong to updates.jenkins.io.  Make sure the url has a certificate that belongs to it or use validate_certs=False (insecure)"}
```

I can get past this by adding `validate_certs=no` to the `get_url` task on line 18 of `tasks/cli.yml`:

```
- name: Get Jenkins updates
  get_url: url=https://updates.jenkins-ci.org/update-center.json dest={{ jenkins.updates_dest }} thirsty=yes mode=0440 timeout=30 validate_certs=no
  register: jenkins_updates
```

I can do a PR for the change, but that doesn't seem like the right fix, as [that cert ought to validate](https://www.sslshopper.com/ssl-checker.html#hostname=updates.jenkins-ci.org).

Let me know if I should open up a different issue for this.

**Comment by clarete** [comment]
Nah, Jenkins people kept the URLs around, that's not the problem. Here's what's going on: https://github.com/ansible/ansible/issues/11579

Just upgrade to a python version newer than 2.7.8 and it will just work.
